icu4j/maven-build/maven-icu4j-datafiles/pom.xml

```diff
@@ -0,0 +1,188 @@
+<?xml version="1.0"?>
+<!--
+* © 2016 and later: Unicode, Inc. and others.
+* License & terms of use: http://www.unicode.org/copyright.html
+*******************************************************************************
+* Copyright (C) 2010-2016, International Business Machines Corporation and    *
+* others. All Rights Reserved.                                                *
+*******************************************************************************
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  
+  <parent>
+    <groupId>com.ibm.icu</groupId>
+    <artifactId>icu4j-root</artifactId>
+    <version>73.1-SNAPSHOT</version>
+    <!-- default relativePath for parent -->
+  </parent>
+  
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.ibm.icu</groupId>
+  <artifactId>icu4j-datafiles</artifactId>
+  <version>73.1-SNAPSHOT</version>
+
+  <packaging>jar</packaging>
+
+  <name>ICU4J Data Files</name>
+  <description>Various *.jar files needed by ICU4J components' main code.</description>
+  <url>https://icu.unicode.org/</url>
+  <inceptionYear>2001</inceptionYear>
```

Comment:
Is this the first year ICU4J got published via Maven? Otherwise it's older than that...

Reply:
@yumaoka